configure.ac

```diff
@@ -289,6 +289,53 @@ else
   AC_SUBST([M32], [0])
 fi
 
+AC_ARG_ENABLE([static_libstdcxx],
+            [AS_HELP_STRING([--enable-static-libstdcxx],
+                            [Compile $CXX with -static-libstdc++.
+                             This allows DMTCP executables compiled on later
+                             distro to run correctly on earlier distros.)])],
+            [use_static_libstdcxx=$enableval],
+            [use_static_libstdcxx=no])
+
+AM_CONDITIONAL(USE_STATIC_LIBSTDCXX, [test "$use_static_libstdcxx" = "yes"])
+if test "$use_static_libstdcxx" = "yes"; then
+  AC_SUBST([STATIC_LIBSTDCXX], [1])
```

Comment:
We don't seem to be using this variable, so why not delete this here and in the else part.